.github/workflows/pr_announcement.yml

```diff
@@ -37,16 +37,29 @@ jobs:
                   addComment(input: {
                           subjectId: ${{ env.PR_ID }},
                           body: """Dear contributor,
-                                  Avocado is currently under sprint #103, which is due to release an LTS (Long Term Stability) release. 
+                                  Avocado is currently at the end of sprint #105, therefore we are in feature freeze state.
                                   Please avoid merging changes that do not fall into these categories:
                                   * Bug fixes
-                                  * Usability Improvements
-                                  * Documentation updates
```

Comment:
Do you see a problem with allowing "Documentation updates"? They will never disrupt stability and are usually along the side of bug fixes.  IMO, we'd allow it.